_includes/get_a_quote.html

```diff
@@ -23,6 +23,11 @@ <h1>Get a quote</h1>
                         <span>Email</span>
                         <input type="text" name="email" placeholder="peter.parker@dailyplanet.com">
                     </label>
+                    <!-- This is a honeypot, it is invisible to users, bot not to bots. If it has value, it means it is a bot -->
+                    <label for="mail" class="honeypot">
+                      <span>mail</span>
+                      <input type="text" name="mail" placeholder="mary.jane@broadwaytheater.com">
+                  </label>
```

Comment:
Indentation weirdness on this line.

Also, I just looked at it visually, and the label has some margin, so it messes with the layout a little bit.

We can fix this by moving this field all the way to the top of the form and setting `style="margin:0;"` on the label :smile: